pinax/stripe/tests/test_actions.py

```diff
@@ -227,6 +227,9 @@ def setUp(self):
             interval_count=1,
             name="Pro"
         )
+        self.account = Account.objects.create(
+            stripe_id="acc_XXX"
+        )
```

Comment:
Could go into a new `CustomersWithConnectTests` (derived) TestCase (with the two test functions).

Reply:
ok